flake.nix

```diff
@@ -99,6 +99,7 @@
           inputsFrom = lib.attrValues inputs.self.checks.${system};
           packages = [
             pkgs.cargo-insta
+            pkgs.cargo-nextest
```

Comment:
Does this mean we should run tests with `cargo nextest` now? If yes, shouldn't that go somewhere in the README at least?

Reply:
You can run the tests with either, it's just that nextest is faster and it captures the stdout.